systemd/cloud-print-connector.service

```diff
@@ -7,13 +7,13 @@
 [Unit]
 Description=Google Cloud Print Connector
 Documentation="https://github.com/google/cloud-print-connector"
-After=cups.service avahi-daemon.service network-online.target
-Wants=cups.service avahi-daemon.service network-online.target
+After=printer.target network.target multi-user.target
```

Comment:
The `After` and `Wants` are correct:
- We rely on CUPS, not printer hardware presence (see https://www.freedesktop.org/software/systemd/man/systemd.special.html)
- `network-online.target` is correct, because we currently will hard-fail unless network connectivity is present at startup (see #160)
- we are `WantedBy=multi-user.target` below, not here
- we have a hard dependency on Avahi
- we need both `After` and `Wants`. `After` is an ordering constraint, not a dependency